src/engine/Server.h

```diff
@@ -134,14 +134,16 @@ class Server {
 
   /// Schedule a task to trigger the timeout after the `timeLimit`.
   /// The returned callback can be used to prevent this task from executing
-  /// either because the `abortionHandle` has been aborted by some other mean
-  /// or because the task has been completed successfully.
+  /// either because the `cancellationHandle` has been aborted by some other
+  /// mean or because the task has been completed successfully.
   static auto cancelAfterDeadline(
       const net::any_io_executor& executor,
-      std::weak_ptr<ad_utility::AbortionHandle> abortionHandle,
+      std::weak_ptr<ad_utility::CancellationHandle> cancellationHandle,
       std::chrono::seconds timeLimit);
 
-  static std::function<void()> setupAbortionHandle(
+  /// Create a cancellation handle, pass it to the operation and configure
+  /// it to get cancelled automatically if a time limit is passed.
+  static std::function<void()> setupCancellationHandle(
```

Comment:
Also document the return time (can refer to the other comment if its the same). Also state which of those functions calls which.